src/templates/static/legal/tac_sections/biel.jsx

```diff
@@ -2,50 +2,92 @@ import React from 'react';
+        <p>{it.L('The client acknowledges and agrees to the following:')}
+            <ul className='bullet'>
+                <li>{it.L('The Company will not pay interest on the Client Money.')}</li>
+                <li>{it.L('Where the Client\'s obligations to the Company are due and payable, the Company shall cease to treat as Client Money the amount equal to the amount of such obligations.')}</li>
+                <li>{it.L('The Company may use the Client Money for the purposes of meeting obligations that are incurred by the Company in connection with the margining, guaranteeing, securing, transferring, adjusting, or settling of the client\'s dealings in derivatives.')}</li>
+            </ul>
+        </p>
 
         <h2 data-anchor='investor-compensation-scheme'>{it.L('Investor Compensation Scheme')}</h2>
-        <p>{it.L('BIEL forms part of [_1]the Investor Compensation Scheme[_2] (the "Scheme") which is a rescue fund for investors that are clients of failed investment firms licensed by the MFSA. The Scheme covers 90% of the Company\'s net liability to a client in respect of investments which qualify for compensation under the Investment Services Act subject to a maximum payment to any one person of €20,000. The Scheme is based on EC Directive 97/9.', '<a href="http://www.compensationschemes.org.mt" target="_blank" rel="noopener noreferrer">', '</a>')}</p>
-        <p>{it.L('Cover is made available on the basis of the depositor rather than on the basis of the number of deposits, meaning that if an individual has multiple accounts he will only be covered as to €20,000 on the global amount. Any other amount exceeding such threshold is not protected and will thus have to be borne by the investor.')}</p>
+        <p>{it.L('BIEL forms part of [_1]the Investor Compensation Scheme[_2] (the "Scheme"), which is a rescue fund for investors that are clients of failed investment firms licensed by the MFSA. The Scheme covers 90% of the Company\'s net liability to a client in respect of investments that qualify for compensation under the Investment Services Act subject to a maximum payment to any one person of €20,000. The Scheme is based on EC Directive 97/9.', '<a href="http://www.compensationschemes.org.mt" target="_blank" rel="noopener noreferrer">', '</a>')}</p>
+        <p>{it.L('Cover is made available on the basis of the depositor rather than on the basis of the number of deposits, meaning that if an individual has multiple accounts, he will only be covered up to €20,000 on the global amount. Any other amount exceeding such a threshold is not protected and will thus have to be borne by the investor.')}</p>
```

Comment:
```suggestion
        <p>{it.L('Cover is made available on the basis of the depositor rather than on the basis of the number of deposits, meaning that if an individual has multiple accounts, they will only be covered up to €20,000 on the global amount. Any other amount exceeding such a threshold is not protected and will thus have to be borne by the investor.')}</p>
```